src/app/portlets/content-types/form/content-types-form.component.ts

```diff
@@ -101,9 +101,12 @@ export class ContentTypesFormComponent implements OnInit, OnDestroy {
         this.name.nativeElement.focus();
 
         if (!this.isEditMode()) {
-            this.dotWorkflowService.getSystem().pipe(take(1)).subscribe((workflow: DotWorkflow) => {
-                this.form.get('workflow').setValue([workflow.id]);
-            });
+            this.dotWorkflowService
```

Comment:
why just in this case?
what should happend if it is EDIT_MODE?

Reply:
EDIT MODE? in content types?